packages/next/server/app-render.tsx

```diff
@@ -1177,7 +1179,7 @@ export async function renderToHTMLOrFlight(
                   <link
                     key={fontFile}
                     rel="preload"
-                    href={`/_next/${fontFile}`}
+                    href={`${assetPrefix}/_next/${fontFile}`}
                     as="font"
                     type={`font/${ext}`}
                     crossOrigin="anonymous"
```

Comment:
Currently, we only load CSS and JS through the `assetPrefix` which doesn't require CORS to be enabled on the separate CDN. However, if we load fonts through the CDN, it would be a breaking change as CORS is now required.

**I add `assetPrefix` for the font here just to match the existing implementation in `pages/_document`**. But IMHO this should be reconsidered.

cc @hanneslund

Reply:
Thanks, good point!